docs/public-networks/reference/cli/options.md

```diff
@@ -3120,7 +3120,16 @@ rpc-gas-cap=50000000
 
 </Tabs>
 
-Sets a limit on the amount of gas for transaction simulation RPC methods. Its value must be greater than or equal to `0`. The default is `0`, which indicates there is no limit. This cap prevents [`eth_call`](../api/index.md#eth_call) requests from using excessive resources.
+Sets a limit on the amount of gas for transaction simulation RPC methods. 
+Its value must be greater than or equal to `0`. 
+The default is `0`, which indicates there is no limit. 
+This cap prevents [`eth_call`](../api/index.md#eth_call) requests from using excessive resources.
+
+:::note
+This option allows users to override the transaction's gas limit. 
```

Comment:
Can we combine this sentence with, "Sets a limit on the amount of gas for transaction simulation RPC methods. " in the opening paragraph? We can then remove it from the note.